src/clr-angular/data/datagrid/render/dom-adapter.ts

```diff
@@ -37,10 +37,26 @@ export class DomAdapter {
         return parseInt(element.getBoundingClientRect().right, 10);
     }
 
+    clientRectLeft(element: any): number {
+        return parseInt(element.getBoundingClientRect().left, 10);
```

Comment:
`getBoundingClientRect` can be expensive, you don't want to call it over and over. It was fine when you called it only once per element for the column resizer, but now you're calling it over and over. Please just expose a single method `clientRect(element: any): ClientRect`, store the result in your directive/component and then get the various properties you need from it.